wpa_supplicant/wpa_cli_cmds.c

```diff
@@ -3013,6 +3013,31 @@ static const struct wpa_cli_cmd wpa_cli_commands[] = {
 	{ "interface_list", wpa_cli_cmd_interface_list, NULL,
 	  cli_cmd_flag_none,
 	  "= list available interfaces" },
+#ifdef CONFIG_AP
```

Comment:
Mainly for memory savings. Using Full fledged AP mode with hostapd adds lots of memory and for the basic use case of Wi-Fi based provisioning basic AP mode should suffice.

Reply:
But basic AP mode does not provide WPA2/3 Enterprise support. Any plan to add hostapd for that?

Reply:
Yes, it doesn't. Support for Enterprise security is in the roadmap, but not ETA as of yet.

Reply:
@gannicus-gj @sandip-dalvi, as we move this hostap along, it can advance in multiple vectors to support the needs of the overall community and individual community members. @krish2718 and @jukkar are following a roadmap but that doesn't mean you can't augment that with changes you need. As long as everything is wrapped by Kconfig then the various needs will be supported.